src/avram/model.cr

```diff
@@ -158,11 +158,13 @@ class Avram::Model
 
   macro setup_getters(columns, *args, **named_args)
     {% for column in columns %}
-      def {{column[:name]}}
-        {% if column[:type].is_a?(Generic) %}
-          {{column[:type].type_vars.first}}::Lucky.from_db! @{{column[:name]}}
+      {% db_type = column[:type].is_a?(Generic) ? column[:type].type_vars.first : column[:type] %}
+      def {{column[:name]}} : {% if column[:nilable] %}::Union({{db_type}}, ::Nil){% else %}{{db_type}}{% end %}
```

Comment:
This one is a bit tricky. `column[:type]` could be `Array(String)`, but `db_type` would be `String`. So I think we could do 

```crystal
def {{column[:name]}} : {% if column[:nilable] %}::Union({{db_type}}, ::Nil){% else %}{{column[:type]}}{% end %}
```

But the `db_type` is still ok, because we need it on the next line so `Array(String)` maps to `String::Lucky.from_db`